<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadRoads"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuZoning">
    <property name="title">
     <string>Zoning</string>
    </property>
    <addaction name="actionGenerateZoningByPM"/>
    <addaction name="actionGenerateManyZoningsByPM"/>
    <addaction name="actionFindBestZoningByPM"/>
    <addaction name="separator"/>
    <addaction name="actionGenerateRandomPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuZoning"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoadRoads">
   <property name="text">
    <string>Load Roads</string>
   </property>
  </action>
  <action name="actionGenerateZoningByPM">
   <property name="text">
    <string>Generate Zoning By PM</string>
   </property>
  </action>
  <action name="actionGenerateManyZoningsByPM">
   <property name="text">
    <string>Generate Many Zonings By PM</string>
   </property>
  </action>
  <action name="actionFindBestZoningByPM">
   <property name="text">
    <string>Find Best Zoning By PM</string>
   </property>
  </action>
  <action name="actionGenerateRandomPreferences">
   <property name="text">
    <string>Generate Random Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
